### PR TITLE
interface: changed ProfileOverlay to functional component

### PR DESCRIPTION
**Pull request by tylershuster**

### CODE REVIEW

File: pkg/interface/src/views/components/ProfileOverlay.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import {
   Icon
 } from '@tlon/indigo-react';
 import RichText from './RichText';
-import { withLocalState } from '~/logic/state/local';
+import useLocalState from '~/logic/state/local';
```
Comment (matildepark) [inline pkg/interface/src/views/components/ProfileOverlay.tsx:17]: shouldn't we be using settings state here? this isn't in localstorage anymore

Comment (tylershuster) [reply pkg/interface/src/views/components/ProfileOverlay.tsx:17]: image

uh should we

Comment (matildepark) [reply pkg/interface/src/views/components/ProfileOverlay.tsx:17]: `state.dark` is changed locally using an eventlistener, so it makes sense in Author. same with toggleOmnibox. but for ViewProfile and this ProfileOverlay it seems worth changing to use `settings.calm` selector stuff.

Comment (tylershuster) [reply pkg/interface/src/views/components/ProfileOverlay.tsx]: Ok I changed this component. Talking with @liam-fitzgerald about the other stuff

**Review by matildepark — APPROVED** [review]
Scope of this PR works, looks fine

**Pull request merged**